docs/airgapped-docs/prereqs.md

```diff
@@ -3,7 +3,14 @@
 In order to utilize Airgapped Docs, you will need the following prerequisites:
 
 - **Rancher v2.7.0 or higher:** https://ranchermanager.docs.rancher.com/integrations-in-rancher/rancher-extensions
-- For fetching and packaging... **Hauler:** https://github.com/rancherfederal/hauler
-- For installation and deployment... **Helm:** https://helm.sh/docs/intro/install
-- Access to the `local` cluster, to be able to install and deploy the app(s).
-- Access to the `downstream` cluster(s), to be able to install and deploy the app(s).
\ No newline at end of file
+- **Helm:** https://helm.sh/docs/intro/install
+- Access to the `downstream` cluster(s), to be able to install and deploy the app(s).
+
+It is also assumed you have followed all of the Carbide Secured Registry (CSR) documentation.
+
+This means you have:
```

Comment:
Possibly add links to each of these steps in the other parts of the docs?

Reply:
I understand wanting to link to respective documentation, but I'd rather leave it the way it is. I almost left it at just `It is also assumed you have followed all of the Carbide Secured Registry (CSR) documentation.`, since it seems unnecessarily redundant to have links in the exact same layout as the sidebar for each additional product/component.